dispatcher/MongoConnect.py

```diff
@@ -393,15 +396,14 @@ def set_stop_time(self, number, detectors, force):
             return
         try:
             time.sleep(0.5) # this number depends on the CC command polling time
-            endtime = self.get_ack_time(detectors, 'stop')
-            if endtime is None:
+            if (endtime := self.get_ack_time(detectors, 'stop') )is None:
```

Comment:
I know this is a walrus, but what does it actually do in this case?

Reply:
Nevermind, got it

Reply:
There's a space missing there anyway